### PR TITLE
Add metrics from Pypi

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements
Introduce metric from PyPI statistics

### CODE REVIEW

File: thoth/metrics_exporter/jobs/pypi.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+
+class PyPIMetrics(MetricsBase):
+    """Class to collect statistics from PyPI."""
+
+    @classmethod
+    @register_metric_job
+    def get_pypi_statistics(cls) -> None:
+        """Get statistics from PyPI."""
+        response = requests.get("https://pypi.org/")
```
Comment (goern) [inline thoth/metrics_exporter/jobs/pypi.py:40]: do we need some exception handling here?

File: thoth/metrics_exporter/jobs/pypi.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        soup = BeautifulSoup(response.content, "html.parser")
+        statistics = soup.findAll("p", class_="statistics-bar__statistic")
+        pypy_stats = [int("".join(c for c in s.get_text() if c.isdigit())) for s in statistics]
+
+        pypi_packages = pypy_stats[0]
```
Comment (goern) [inline thoth/metrics_exporter/jobs/pypi.py:45]: this should have some sane defaults

File: thoth/metrics_exporter/jobs/pypi.py (hunk 1 above)
Comment (pacospace) [reply thoth/metrics_exporter/jobs/pypi.py:40]: we run each metric as
```
    try:
        job()
    except Exception:
        _LOGGER.exception("Failed to run metrics gathering job %s.%s", class_name, method_name)
```

File: thoth/metrics_exporter/jobs/pypi.py (hunk 2 above)
Comment (pacospace) [reply thoth/metrics_exporter/jobs/pypi.py]: Addressed in https://github.com/thoth-station/Qeb-Hwt/pull/48

File: thoth/metrics_exporter/jobs/pypi.py (hunk 1 above)
Comment (fridex) [reply thoth/metrics_exporter/jobs/pypi.py:40]: To have correct reports:

```python
response.raise_for_status()
```

**Comment by pacospace** [comment]
> LGTM, just one minor thing

Updated in f01dd7db169c5def13470fca8df5dcb0095a62ac

**Comment by fridex** [comment]
recheck

**Pull request merged**